opm/simulators/wells/MultisegmentWell_impl.hpp

```diff
@@ -411,21 +411,10 @@ namespace Opm
         const auto& group_state = ebosSimulator.problem().wellModel().groupState();
         auto& ws = well_state_copy.well(this->index_of_well_);
 
-        // Get the current controls.
-        const auto& summary_state = ebosSimulator.vanguard().summaryState();
-        auto inj_controls = well_copy.well_ecl_.isInjector()
```

Comment:
This change also makes the correctness of this PR in question or a bug in the master branch.